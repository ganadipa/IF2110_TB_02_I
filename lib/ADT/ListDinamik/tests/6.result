lib/ADT/ListDinamik/tests/6.in lib/ADT/ListDinamik/tests/6.out: PASSED!
