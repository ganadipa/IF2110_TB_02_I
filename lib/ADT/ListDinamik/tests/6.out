Max: 5, Min: 1
